app/controllers/landlords_controller.rb

```diff
@@ -0,0 +1,36 @@
+class LandlordsController < ApplicationController
+  def create
+  	if current_user.type == 'Landlord'
```

Comment:
This would make it so only Landlords can create Landlords, which isn't necessarily true.